bundles/pixi.js-node/src/adapter/adapter.ts

```diff
@@ -36,7 +36,7 @@ export const NodeAdapter = {
 
         return new Promise((resolve, reject) =>
         {
-            const filePath = utils.path.normalize(request.url);
+            const filePath = utils.path.normalize(typeof url === 'string' ? url : request.url);
```

Comment:
My only suggestion is to add a comment, since reading this it's not super obvious why we'd want to use url. Something like:

```
// Request transforms paths and encodeURIs, but for filesystem requests, it's better to use the raw string (path)
```